backend/controllers/movieController.js

```diff
@@ -75,12 +78,13 @@ const addComment = async (req, res) => {
 const playMovie = async (req, res, next) => {
   const { imdbCode } = req.params;
   let movie = await Movie.findOne({ imdbCode });
-  if (!movie || !movie.downloadComplete) {
+  if ((!movie || !movie.downloadCompleted) && !downloadCache.has(imdbCode)) {
```

Comment:
Seems very strange! Have you checked the network tab in the developer tools to see if you receive the 20mb videostream chunks?

Reply:
I checked today. I am receiving 206, and it's size is not stable. First response grows until 20 MB, and next one is freeze somewhere about 6-8 MB.
Each movie is playing only first 3-10 minutes.
Could @iljaSL , @T7Q , @lapaset also check this one? Do you have this problem? (I can not watch the whole movie, it always stops somewhere in 3-10 minutes after play, even if movie is loaded completely).

Reply:
Same issue here. I tried different videos and the result varies. For me it can be also the first chunk that is less than 20MB when play starts. Here are my observations:
- If the loaded chunk is below 20MB the video stops when it's reached and loading seems to stop in the network tab
-> If I start the video again it will not get beyond that same spot
-> However when I open the video in my local videoplayer it has the parts that are missing
- When I refresh and play the video again it may or may not load the missing parts
- Sometimes after a couple restarts 20MB chunks start loading. In that case everything works smoothly
- Video player shows the file as fully downloaded even when it is not
-> Could the problem be that it is trying to play something that does not exist?
- VSCode shows the video size as the size of the complete file even when it is not loaded

Reply:
Interesting. As a simple fix we could just remove the chunk functionality completely so the backend would just stream the whole video back-to-back without splitting it into 20mb pieces.

Reply:
Made a [branch](https://github.com/DianaMukaliyeva/hypertube/tree/backend/remove_chunk) with no chunking and a new issue, #106